tests/cli/test_install.py

```diff
@@ -129,6 +130,26 @@ def test_install_family(run_cli_command, get_pseudo_archive):
     assert len(family.pseudos) != 0
 
 
+pytest.mark.usefixtures('clear_db')
+
+
+def test_install_family_folder(run_cli_command, filepath_pseudos):
+    """Test ``aiida-pseudo install family` from folder`."""
+    label = 'family_test'
+    description = 'description'
+    dirpath = filepath_pseudos()
+    options = ['-D', description, dirpath, label]
+
+    result = run_cli_command(cmd_install_family, options)
+    assert f'installed `{label}`' in result.output
+    assert PseudoPotentialFamily.objects.count() == 2
```

Comment:
why should this be 2? The database is cleared before this test isn't it?

Reply:
Still don't really understand why this should be 2? Maybe because this test is not properly decorated (because of the empty lines after the decorator with the fixture `clean_db`)

Reply:
Sorry I forgot to check about this. Will do now. All the rest has been addressed.

Reply:
The decorator was misplaced. I'll push now the fix. Sorry!